<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>toolbar_rendertransparencygroup</class>
 <widget class="QWidget" name="toolbar_rendertransparencygroup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>167</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Render Transparency Group</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>3</number>
   </property>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="checkBox_negativeEffect">
     <property name="text">
      <string>Negative Effect</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QSpinBox" name="spinBox_transparency">
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSlider" name="slider_transparency">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="checkBox_enhanceContrast">
     <property name="text">
      <string>Enhance Contrast</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="checkBox_transparency">
     <property name="text">
      <string>transparency</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" rowspan="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>checkBox_transparency</tabstop>
  <tabstop>checkBox_enhanceContrast</tabstop>
  <tabstop>checkBox_negativeEffect</tabstop>
  <tabstop>slider_transparency</tabstop>
  <tabstop>spinBox_transparency</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
